src/get-started/wordcount-example.md

```diff
@@ -166,16 +195,25 @@ Pipeline p = Pipeline.create(options);
 
 ### Applying Pipeline Transforms
 
-The Minimal WordCount pipeline contains several transforms to read data into the pipeline, manipulate or otherwise transform the data, and write out the results. Each transform represents an operation in the pipeline.
+The Minimal WordCount pipeline contains several transforms to read data into the
+pipeline, manipulate or otherwise transform the data, and write out the results.
+Each transform represents an operation in the pipeline.
```

Comment:
I don't like the use of the word 'operation' here - it's some transform over the data, but it may be implemented in terms of multiple other transforms. Some basic transforms represent an individual operation (I'm understanding operation to be synonymous with 'primitive', which it may not be as you use it)

Reply:
done